src/utils/weather.js

```diff
@@ -1,13 +1,19 @@
 import axios from 'axios';
 
-export const currentWeather = async () => {
-  const url = `http://api.openweathermap.org/data/2.5/weather?lat=30.267153&lon=-97.7430608&appid=${
-    process.env.OPENWEATHER_APIKEY
-  }&units=imperial`;
-
+export const getWeatherInformation = async url => {
   const response = await axios.get(url);
-
   return response;
```

Comment:
you can just return the axios call. it'll be wrapped in a promise either way.

Reply:
done